modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -351,6 +368,13 @@ var CBIWifiFrequencyValue = form.Value.extend({
 				],
 				'ac': [
 					'11a', '5 GHz', true
+				],
+				// 'ad': [
+				//      '?', '60 GHz', true
```

Comment:
Upstream hostapd.conf would use `hw_mode=ad` to select 60GHz, so in OpenWrt uci it'd likely get exposed as `option hwmode 11ad`